index.html

```diff
@@ -1,37 +1,39 @@
         <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
         <link rel="stylesheet" href="./style.css" />
+        <script src="./script.js" defer></script>
         <title>Aron Abraham Portfolio</title>
     </head>
     <body>
-
-        <!-- Header of the page with navigation bar -->
-        <header class="top-navigation">
-            <a href="#" class="log">Aron Abraham</a>
-            <nav class="item-navigation">
-                <ul class="frame-navigation">
-                    <li class="item-frame-nav"><a href="#portfolio-link">Portfolio</a></li>
-                    <li class="item-frame-nav"><a href="#about-link">About</a></li>
-                    <li class="item-frame-nav"><a href="#contact-link">Contact</a></li>
-                    <li class="item-frame-nav"><a href="#email-link" aria-label="email envelope"><i class="fa fa-envelope-o" ></i></a></li>
-                </ul>
-                <a href="#icon-bar-link"  class="icon-bar"><img src="./image/Vector-1.svg" alt="Union image bar"/></a>
-            </nav>
-        </header>
         <div class="main">
-
-            <!-- Primary text headline -->
+            <header class="item-navigation">
+                <div class="log">Aron Abraham</div>
+                <nav class="navigation">
+                    <div class="hamburger-menu">
+                        <span class="bar"></span>
+                        <span class="bar"></span>
+                        <span class="bar"></span>
+                    </div>
+                    <ul data-visible="false" class="frame-navigation">
+                        <li class="close-menu"><span class="material-symbols-outlined">close</span></li>                        
+                        <li class="item-frame-nav"><a href="#portfolio" class="nav">Portfolio</a><span class="material-symbols-outlined chevron-right">chevron_right</span></li>
+                        <li class="item-frame-nav"><a href="#about" class="nav">About</a><span class="material-symbols-outlined chevron-right chevron-right-about">chevron_right</span></li>
+                        <li class="item-frame-nav"><a href="#contact" class="nav">Contact</a><span class="material-symbols-outlined chevron-right">chevron_right</span></li>
+                        <li class="item-frame-nav"><a href="#email" class="link" aria-label="email envelope"><i class="fa fa-envelope-o"></i></a></li>
+                    </ul>
+                </nav>
+            </header>
+            <!-- Primary text headline-->
             <section class="container-header">
                 <h1 class="header-text">Aron Abraham</h1>
                 <h2 class="support-text">Software Developer</h2>
```

Comment:
- [ **OPTIONAL** ] **IMPORTANT && RECOMENDED**: Kindly make sure all the navigation links(Portfolio, About, Contact) are visible in all desktop screen sizes to make your portfolio professional. I noticed that some of them disappear on medium screen size, imagine a user using a tablet and how he can access navigation items options when they are not visible. Also, make sure to remove the extra space on the right and make your project responsive
#### Here is how it looks: 
![nav_iss](https://user-images.githubusercontent.com/83042742/174698594-735c444b-426c-45e0-ab8c-2670a6f04a57.png)